src/content/sessions/2024-05-10.md

```diff
@@ -0,0 +1,28 @@
+﻿---
+title: "F# Tree-Sitter"
+preview: "F# Tree-Sitter"
+isDraft: true
+date: 2024-05-10T13:00:00.000Z
+slug: "2024/05/10"
+champion: "Vlad Zarytovskii"
+zoomLink: ""
+zoomPasscode: "tree"
+issueLink: ""
+company: "Microsoft"
+youtubeId: ""
+---
+
+## Topic
+
+Join us for a session on F# Tree-Sitter. We will look into [F# Tree-Sitter](https://github.com/ionide/tree-sitter-fsharp) and how it can be used to improve the F# language service across different editors.
```

Comment:
I would assume a lot of people are not familiar at all with tree sitter.
Perhaps include https://tree-sitter.github.io/tree-sitter/ or https://www.youtube.com/watch?v=Jes3bD6P0To&ab_channel=StrangeLoopConference as well.

Reply:
Good idea 👍